Parcelable error when reloading app back from stack and has been cleared by the os

Caused by: java.lang.RuntimeException: Parcel android.os.Parcel@1b179a5d: Unmarshalling unknown type code 7471204 at offset 1112

STR:
Enable Don't keep activities in Developer options. Start the app and go to activity with the rc progressbar.
Get out of app and go back to it. crash.

From what i gathered 
Happens due to progaurd

Need to add

-keepclassmembers class \* implements android.os.Parcelable {
    static *\* CREATOR;
}

in progaurd-rules

--- a/library/src/main/java/com/akexorcist/roundcornerprogressbar/common/BaseRoundCornerProgressBar.java
+++ b/library/src/main/java/com/akexorcist/roundcornerprogressbar/common/BaseRoundCornerProgressBar.java
@@ -512,6 +512,10 @@ public abstract class BaseRoundCornerProgressBar extends LinearLayout {
             out.writeInt(this.secondaryProgressColor);
             out.writeInt(this.backgroundColor);
 
+            out.writeInt(this.layoutBackgroundColor);
+            out.writeInt(this.layoutProgress);
+            out.writeInt(this.layoutSecondaryProgress);
+
             out.writeByte((byte) (this.isProgressBarCreated ? 1 : 0));
             out.writeByte((byte) (this.isProgressSetBeforeDraw ? 1 : 0));
             out.writeByte((byte) (this.isMaxProgressSetBeforeDraw ? 1 : 0));

